src/python/pants/option/config.py

```diff
@@ -186,6 +197,7 @@ class _ConfigValues(ABC):
   in the future if we ever decide to support formats other than INI.
```

Comment:
Might make sense to update this comment now that we're supporting formats other than INI

Reply:
Good catch!